### DISCUSSION TITLE
Are there examples using electromagnetic fields?

### DISCUSSION THREAD

**Discussion by 1035326373** [nrc-cnrc/EGSnrc#917] [Questions] [answered]
Are there any used_codes examples of electromagnetic fields? I checked `emf_ macros.mortran`. It seems that the implementation of this file is incomplete or inconsistent with the introduction of pirs701.

**Comment by 1035326373** [comment, marked as answer]
Further, what are the similarities and differences between the handling of magnetic fields in MCNP6 and ESGnrc, including macros `EEMF_macros.mortran` and `emf_macros.mortran`?

**Reply by rtownson** [reply]
You can add magnetic and electric fields to any code by adding lines to the transport parameters:

```
:start MC transport parameter:
Magnetic Field = 0,15,0
Electric Field = 10,0,0
EM ESTEPE = 0.2
:stop MC transport parameter:
```

Personally I haven't looked into magnetic fields in MCNP6. Historically EGSnrc has more reliable electron transport and passes the Fano test, but I'm not sure how the most recent versions of MCNP have changed.
